engines/dfc_provider/app/controllers/dfc_provider/api/products_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# Controller used to provide the API products for the DFC application
+module DfcProvider
+  module Api
+    class ProductsController < ::ActionController::Metal
```

Comment:
I'm wondering if this whole controller could be a lot cleaner if it inherited from `ActionController::Base` instead of `Metal`. Basically `ActionController::Metal` is a stripped-down version of `ActionController::Base`, with all of these bits like `ActionController::MimeResponds` removed.

What do you think?

Reply:
Yes actually, working fine with it.